Files/Views/LayoutModes/GridViewBrowser.xaml.cs

```diff
@@ -167,14 +167,6 @@ protected override void OnNavigatedTo(NavigationEventArgs eventArgs)
 
         protected override void OnNavigatingFrom(NavigatingCancelEventArgs e)
         {
-            var selectorItems = new List<SelectorItem>();
-            DependencyObjectHelpers.FindChildren<SelectorItem>(selectorItems, FileList);
-            foreach (SelectorItem gvi in selectorItems)
-            {
-                base.UninitializeDrag(gvi);
-                gvi.PointerPressed -= FileListGridItem_PointerPressed;
-            }
-            selectorItems.Clear();
```

Comment:
This block was removed because we now uninitialize item drag during (recycle) unload.